btree/btree.c

```diff
@@ -3121,7 +3139,10 @@ static int64_t btree_put_root_split_handle(struct m0_btree_op *bop,
 	int curr_max_level = node_level(lev->l_node);
 
 	/* skip the invarient check for level */
-	oi->i_extra_node->n_skip_rec_count_check   = true;
+	n_lock_op_lock(lev->l_node);
+	n_lock_op_lock(oi->i_extra_node);
```

Comment:
This is newly allocated node and any other kv_op does not know about this node yet. Do we need to take for `i_extra_node`?

Reply:
Yes , other kv_op will not know about this at this point but it i going to get attached at  root node at line.3189, so then kv_op may try to access it. Therefore doing lock unlock for exra_node as well. Moreover, as extra node will be new node, it will get lock quickly.

Reply:
Ok.